securedrop/requirements/python3/develop-requirements.in

```diff
@@ -11,7 +11,7 @@ docker
 dnspython
 html-linter
 importlib-resources
-markupsafe>=2.0
+markupsafe==2.1.2
```

Comment:
I would prefer not to pin in the `.in` file because we don't actually want to pin it, we just want it to match. I'll amend shortly.